src/oas/index.ts

```diff
@@ -1 +1,2 @@
 export * from './operation';
+export { translateSchemaObject } from './transformers/index';
```

Comment:
Do we also need to export `translateLogo`?

Reply:
nope, not used publicly